ftdetect/hcl.vim

```diff
@@ -1,8 +1,13 @@
 autocmd BufNewFile,BufRead *.hcl set filetype=hcl
 
 " Nomad
-autocmd BufNewFile,BufRead *.nomad set filetype=hcl
+autocmd BufNewFile,BufRead *.nomad setfiletype hcl
+
+autocmd FileType nomad set syntax=hcl
 
 " Terraform
-autocmd BufNewFile,BufRead *.tf     set filetype=hcl
-autocmd BufNewFile,BufRead *.tfvars set filetype=hcl
+autocmd BufNewFile,BufRead *.tf     setfiletype hcl
+autocmd BufNewFile,BufRead *.tfvars setfiletype hcl
```

Comment:
Filetype is detected by this plugin
So I think it should be something like this:
```suggestion
autocmd BufNewFile,BufRead *.tf     setfiletype terraform
autocmd BufNewFile,BufRead *.tfvars setfiletype terraform
```

Reply:
Alright, got it. I initially thought that the problem was this plugin overwriting an earlier setting of the file type `terraform`. I updated the code accordingly. 🙂